src/RestControllers/PatientRestController.php

```diff
@@ -19,6 +19,24 @@ class PatientRestController
 {
     private $patientService;
 
+    /**
+     * White list of patient search fields
+     */
+    private const SUPPORTED_SEARCH_FIELDS = array(
+        "fname",
+        "lname",
+        "ss",
+        "street",
+        "postal_code",
+        "city",
+        "state",
+        "phone_home",
+        "phone_biz",
+        "phone_cell",
+        "email",
+        "DOB"
+    );
+
     public function __construct($pid)
```

Comment:
I didn't want to tackle this in my first PR, but are there any thoughts on removing the dependency on the `$pid` in the constructor? We could just pass the $pid in as a parameter via the Rest Controller configuration. Just bringing this up as this seems to be the only controller which uses this technique/pattern.

Reply:
Yep, it can be moved out of constructor to getOne($pid). Then service modified.

Reply:
Thanks! I'll take a look at that change in the next iteration (shining up the patient FHIR endpoint)